src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+
+namespace NuGet.VisualStudio.Internal.Contracts
+{
+    public interface INuGetProjectManagerService : IDisposable
+    {
+        ValueTask<IReadOnlyCollection<PackageReference>> GetInstalledPackagesAsync(IReadOnlyCollection<string> projectGuids, CancellationToken ct);
```

Comment:
@zivkan used Guid in his API, if easier we could consider that here for consistency.

Personally not too invested either, just want different people to be aware of the discrepancies.